.github/workflows/project-codeflare-release.yml

```diff
@@ -152,6 +156,19 @@ jobs:
       shell: bash
       if: ${{ env.SDK_RELEASE_STATUS_CODE != '200' }}
 
+    - name: Sync ODH Notebooks
```

Comment:
Invalid workflow file: .github/workflows/project-codeflare-release.yml#L1
you may only define up to 10 `inputs` for a `workflow_dispatch` event